cmd/build-kube.go

```diff
@@ -11,6 +11,8 @@ var (
 	flagBuildKubeOutputDir       string
 	flagBuildKubeDefaultEnvFiles []string
 	flagBuildKubeUseMemoryLimits bool
+	flagBuildKubeCreateHelmChart bool
```

Comment:
It is just a variant of `fissile build kube`. Having separate command would be a lot of code duplication. But maybe that is ok, if we eventually get rid of `build kube`.